org-code-javabuilder/theater/src/main/java/org/code/theater/Stage.java

```diff
@@ -9,8 +9,9 @@
 import java.io.ByteArrayOutputStream;
 import java.io.FileNotFoundException;
 import java.util.HashMap;
-import org.code.media.AudioWriter;
+import org.code.media.*;
 import org.code.media.Color;
+import org.code.media.Font;
 import org.code.media.Image;
```

Comment:
If we're using a * import, do we also need Color, Font, and Image enumerated separately?

Reply:
yep, the reason is that Color, Font and Image are all defined in `java.awt.*`. By enumerating them separately here we use a nice Java feature in which the class that is specifically imported is the one used by default. So in `Stage` if we reference `Color` (or Font or Image) it will use the `org.code.media` one instead of the `java.awt` one.

Reply:
👍